### PR TITLE
Feat/la admin report

### PR DESCRIPTION
**Pull request by g-m-taylor**
#### Tests
Does this PR include tests for the changes introduced?
- [ x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/test/unit/utils/fileLock.spec.js [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+      });
+
+      // Act
+      const req = httpMocks.createRequest({
+        method: 'GET',
+        url: '/api/reports/localAuthority/establishment/5c03fe27-c69a-4d6c-a740-1f7b5d4b9dbe/user/',
+        establishmentId: 2292
+      });
+      const res = httpMocks.createResponse();
+      await lockFile.acquireLock('la', {}, false, req, res);
```
Comment (chadtomkiss-madetech) [inline server/test/unit/utils/fileLock.spec.js:48]: I am so proud :joy:

File: server/utils/fileLock.js [hunk 2]
```diff
@@ -10,167 +10,197 @@ const Bucket = String(config.get('bulkupload.bucketname'));
...
-      throw new Error('Response code was not numeric');
-    }
-  } catch (err) {
-    console.error('Report::responseGet: getting data returned an error:', err);
-    res.status(404).send({
-      message: 'Not Found'
-    });
-  }
-};
+      IDLockOn = req.establishmentId;
```
Comment (chadtomkiss-madetech) [inline server/utils/fileLock.js:182]: There's `IDLockOn = req.query.subEstId || req.establishmentId;` in the code - does that matter for this `IDLockOn`?

Comment (g-m-taylor) [reply server/utils/fileLock.js:182]: good question, I will double check that

Comment (g-m-taylor) [reply server/utils/fileLock.js]: ok, I added it back in for consistency.  Good catch

**Pull request merged**